code/kildishev-maksim/src/test/java/ru/unn/agile/figuresvolumecalculator/FiguresVolumeCalculatorTest.java

```diff
@@ -0,0 +1,106 @@
+package ru.unn.agile.figuresvolumecalculator;
+
+
+import org.junit.Test;
+import ru.unn.agile.figuresvolumecalculator.model.FiguresVolumeCalculator;
+
+import static org.junit.Assert.*;
+
+
+public class FiguresVolumeCalculatorTest {
+
+    private final double delta = 1e-2;
+
+    @Test
+    public void canCalculatePyramidVolume() {
+        assertEquals(3, FiguresVolumeCalculator.pyramidVolumeCalculate(3, 3), delta);
+    }
```

Comment:
You use a lot of magic numbers, like here, fix it please

Reply:
Fixed!